54,-68-nft-shape/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "nft-shape",
+  "version": "protocol-squad",
+  "description": "SDK7 Template scene",
+  "scripts": {
+    "build": "sdk-commands build --skip-install",
+    "deploy": "sdk-commands deploy",
+    "start": "sdk-commands start"
+  },
+  "devDependencies": {
+    "@dcl/js-runtime": "7.3.36-7291127796.commit-b299b0d",
```

Comment:
commit the same `package.json` placed in the other tests

Reply:
fixed